architecture/core_concepts/pods_and_services.adoc

```diff
@@ -160,6 +161,57 @@ link:https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-gui
 has details about the functionality and purpose of pods.
 ====
 
+[[init-containers]]
```

Comment:
All section titles must have a unique section anchor. `[[init-containers]]` is also used in `architecture/core_concepts/containers_and_images.adoc`